sympy/solvers/ode/tests/test_single.py

```diff
@@ -688,7 +699,8 @@ def _get_all_examples():
     _get_examples_ode_sol_euler_var_para(),
     _get_examples_ode_sol_factorable(),
     _get_examples_ode_sol_bernoulli(),
-    _get_examples_ode_sol_nth_algebraic()]
+    _get_examples_ode_sol_nth_algebraic(),
+    _get_examples_ode_sol_riccati()]
```

Comment:
Next time you add something here it would be good to add a trailing comma. It's useful because then in the diff you only see the lines that are actually changed and you can easily delete or insert lines without editing the others e.g.:
```
a = [
   b,
   c,
]
```
```